assemblies/assembly-add-custom-app-file-openshift.adoc

```diff
@@ -0,0 +1,24 @@
+[id='assembly-add-custom-app-file-openshift_{context}']
+= Adding a custom application configuration file to {ocp-brand-name}
+
+To access the {product}, you must add a custom application configuration file to {ocp-brand-name}. In {ocp-short}, you can use the following content as a base template to create a ConfigMap named `app-config-rhdh`:
+
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  app-config-rhdh.yaml: |
+    app:
+      title: Red Hat Developer Hub
+----
+
+You can add the customer application configuration file to {ocp-short} in one of the following ways:
```

Comment:
above mentioned as "custom (not customer) application configuration file".

Reply:
Good catch!